news/2023-01-07-new-featured-artist-fellowship.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "New Featured Artist: Fellowship"
+date: 2023-01-07 11:00:00 +0000
+---
+
+We found something you've been seeking.
+
+![](https://assets.ppy.sh/artists/332/header.jpg)
+
+Ever since [Glory Days](https://osu.ppy.sh/beatmapsets/1781620) made it to the ranked section, **Fellowship** was prophesized to join our Featured Artist catalogue.
```

Comment:
since rest of article has british spelling

```suggestion
Ever since [Glory Days](https://osu.ppy.sh/beatmapsets/1781620) made it to the ranked section, **Fellowship** was prophesised to join our Featured Artist catalogue.
```